src/index.css

```
@@ -137,3 +138,7 @@
 .focus-mode .tiptap *:not(.has-focus)::marker {
   @apply text-muted-foreground/50;
 }
+
+.filler-word {
```

```suggestion
.tiptap .filler-word {
```

or

```suggestion
.obw-filler-word {
```

`obw` being acronym `one-blog-writer`